worlds/rabi_ribi/client/memory_io.py

```diff
@@ -122,7 +123,7 @@ def _read_4_byte_bool(self, offset):
         :returns: The data represented as a float.
         """
         data = self._read_word(offset)
-        if (struct.unpack("i", data)[0] == 0):
+        if (struct.unpack("<i", data)[0] == 0):
```

Comment:
What's the motivation behind this change? iirc RR does use little endian for its byte interpretation (e.g. in _read_int above).

Reply:
Ah, I'll remove that; I had a mistaken understanding of how to use the `struct` module for a bit.